### PR TITLE
bamPEFragsize improvement

### PR DESCRIPTION
**Pull request by vivekbhr**
I tried to add plotTitle param, hope it works in galaxy..
modified explanation

### CODE REVIEW

File: galaxy/wrapper/computeMatrix.xml [hunk 1]
```diff
@@ -25,6 +25,9 @@
             @THREADS@
 
             #if $output.showOutputSettings == "yes"
+                #if $output.saveData:
```
Comment (bgruening) [inline galaxy/wrapper/computeMatrix.xml]: @vivekbhr is this a intended modification or a bad merge? I think @dpryan79 removed this in an other PR today.

Comment (dpryan79) [inline galaxy/wrapper/computeMatrix.xml]: That was just an oversight from @vivekbhr :P

Comment (vivekbhr) [inline galaxy/wrapper/computeMatrix.xml]: @bgruening My fault.. I replaced this while trying to fix merge conflict

Comment (bgruening) [inline galaxy/wrapper/computeMatrix.xml]: Don't worry this is a lot of fun for me, even if I have holidays :)

**Comment by bgruening** [comment]
:+1:

**Pull request merged**